Tests/ApolloTests/UploadTests.swift

```diff
@@ -198,6 +199,225 @@ class UploadTests: XCTestCase {
     }
     
     self.wait(for: [expectation], timeout: 10)
+  }
+  
+  // MARK: - UploadRequest
```

Comment:
These tests were moved over from the `RequestCreator` tests since they no longer have anything to do with the request creator